images/build/Dockerfile

```diff
@@ -33,17 +33,14 @@ RUN make clean $MAKE_TARGET SOURCE_GIT_TAG=$SOURCE_GIT_TAG
 # -v /lib/modules:/lib/modules \
 # -v /etc:/etc \
 # -v /run/containers:/run/containers \
-# -v /var/log:/var/log   microshift
+# -v /var/log:/var/log
 
 
-FROM registry.access.redhat.com/ubi8/ubi-minimal$DIGEST
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.4
 
 ARG ARCH=amd64
 
-RUN microdnf install -y \
-      policycoreutils-python-utils \
-      iptables && \
-      microdnf clean all
+RUN microdnf install -y policycoreutils-python-utils iptables &&  microdnf clean all
```

Comment:
Does the preview syntax create any kind of error?

Reply:
Not sure what you mean by preview syntax.  This line doesn't produce errors for me on fed33/podman.

Reply:
sorry, I meant previous syntax. What's the purpose of changeing it to a oneliner?

Reply:
None other than to shorten the file - it wasn't the longest line in there.